client/services/code_preprocessors/PythonCodePreprocessorServiceSpec.js

```diff
@@ -190,5 +190,25 @@ describe('PythonCodePreprocessorService', function() {
         }).toThrow(new Error('Incomplete line: missing "(" in def statement.'));
       }
     );
-  })
+
+    it('should add correct performance test information to skeleton code',
+      function() {
+        var performanceTests = [
+          {
+            "input_data_atom": "na ",
+            "transformation_function": "extendString",
+            "expected_performance": "linear",
+            "evaluation_function": "katamariDamashi"
+          }
+        ];
+
+        outputCode = PythonCodePreprocessorService.generatePerformanceTestCode(
+                performanceTests);
+
+        expect(outputCode).toContain('katamariDamashi');
```

Comment:
That's fair, and I agree with the maintainability concern.

For now, can we go mid-way: assert containment, but include a longer portion of the part containing it? That way we know it's in roughly the right place/context.